### DISCUSSION TITLE
Too many connections Issue in Laravel Vapor when regenerating thumbnails

### DISCUSSION THREAD

**Discussion by prakhargvp** [spatie/laravel-medialibrary#3602] [Q&A]
`php artisan media-library:regenerate`

![CleanShot 2024-05-01 at 11 26 18](https://github.com/spatie/laravel-medialibrary/assets/7056772/e2abc524-6b4c-484b-b3d8-c7bcd899356b)

![CleanShot 2024-05-01 at 11 26 44](https://github.com/spatie/laravel-medialibrary/assets/7056772/b7aa0306-5423-4e40-8800-fd8c54d58f16)

**Comment by numeralsix** [comment]
Set some limit on queue concurrency in your vapor.yml

e.g.         queue-concurrency: 50

Make sure this number is lower than your maximum number of database connections

Vapor sees a bunch of jobs in the queue and launches lambdas to handle them. It has no idea when it might exceed your db connection limit
